### PR TITLE
CircleCI - Remove Bento

### PR DESCRIPTION
**Pull request by dmarcoux**
A lot of files were changed and to make this easier to review, I did my best to split everything per commit and add details on why/how I've done what I've done.

I didn't move the feature specs out of `spec/bootstrap/features` since it would involve more work and I don't want to delay this PR any longer since it's blocking us.

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Remove bento from circle-ci](https://trello.com/c/7fshtl2L/1236-remove-bento-from-circle-ci)

**Comment by vpereira** [comment]
Looking into

**Comment by saraycp** [comment]
Great job!! :bouquet:

File: src/api/spec/cassettes/Webui_ProjectController/GET_create/with_a_param_called_maintenance_project/it_should_behave_like_a_valid_project_saved/1_20_2_2_2.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://backend:5352/source/home:tom:my_project/_project/_history?deleted=1
```
Comment (coolo) [inline src/api/spec/cassettes/Webui_ProjectController/GET_create/with_a_param_called_maintenance_project/it_should_behave_like_a_valid_project_saved/1_20_2_2_2.yml]: I'm curious: what are all these _history calls about? it seems to happen everywhere

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 2]
```diff
@@ -207,7 +207,7 @@
       end
 
       it { expect(controller).to set_flash[:error] }
-      it { expect(response).to redirect_to(package_edit_path(project: source_project, package: source_package)) }
+      it { expect(response).to redirect_to(package_show_path(project: source_project, package: source_package)) }
```
Comment (DavidKang) [inline src/api/spec/controllers/webui/package_controller_spec.rb]: The commit description is outdated, apart from that  great job :clap:

Comment (dmarcoux) [reply src/api/spec/controllers/webui/package_controller_spec.rb]: Oops, I forgot to update it!

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 3]
```diff
@@ -394,60 +394,6 @@
...
-      before do
-        allow(Backend::Api::BuildResults::Status).to receive(:result_swiss_knife).and_return(fake_build_results_without_binaries)
-        get :binaries, params: { package: source_package, project: source_project, repository: repo_for_source_project.name }
-      end
-
-      it { expect(response).to have_http_status(:success) }
-      it { expect(response.body).to match(/No built binaries/) }
-    end
-
-    context 'with build results and binaries' do
```
Comment (eduardoj) [inline src/api/spec/controllers/webui/package_controller_spec.rb]: As we remove this tests, we could also remove its cassette and the complete subdirectory: `spec/cassettes/Webui_PackageController/GET_binaries/`.

**Comment by vpereira** [comment]
Please @openSUSE/open-build-service review it!

**Comment by coolo** [comment]
e.g. the spec file still excludes bootstrap

**Comment by DavidKang** [comment]
I think, we also need to run the spider test with bootstrap

**Comment by coolo** [comment]
> I think, we also need to run the spider test with bootstrap

Yes, but that can be a different PR I would say

**Comment by DavidKang** [comment]
> Yes, but that can be a different PR I would say

Yes, I think the dame.

**Comment by hennevogel** [comment]
@coolo please review again

File: .circleci/config.yml [hunk 4]
```diff
@@ -104,17 +104,16 @@ jobs:
...
             cd src/api
             pickfile=log/pick.$CIRCLE_NODE_INDEX.list
-            # reserve the latest node for bootstrap
-            if test "$CIRCLE_NODE_INDEX" = $((CIRCLE_NODE_TOTAL-1)); then
-              echo "Testing bootstrap"
-              circleci tests glob 'spec/features/**/*_spec.rb' 'spec/bootstrap/**/*_spec.rb' > $pickfile
-              export BOOTSTRAP=1
-            else
-              # single out bootstrap and db
-              circleci tests glob 'spec/**/*_spec.rb' | grep -v 'spec/\(bootstrap\|db\)' > spec.list
```
Comment (coolo) [inline .circleci/config.yml]: I overlooked this in my review, but you dropped the db tests now

File: .circleci/config.yml [hunk 5]
```diff
@@ -104,17 +104,16 @@ jobs:
...
+
+            # single out db specs
+            circleci tests glob 'spec/**/*_spec.rb' | grep -v 'spec/db/' > spec.list
+            circleci tests split --total $CIRCLE_NODE_TOTAL --split-by=timings < spec.list > $pickfile
+
             bundle exec rspec --format progress --format RspecJunitFormatter -o /home/frontend/rspec/rspec.xml $(cat $pickfile)
+
             # run DB tests at the end of node 0
             # see https://github.com/openSUSE/open-build-service/issues/4959
             if test "$CIRCLE_NODE_INDEX" = 0; then
```
Comment (coolo) [inline .circleci/config.yml]: ah, there they are - thanks @eduardoj

**Pull request merged**